python-test/features/steps/local_agent.py

```diff
@@ -84,6 +89,7 @@ def run_container_using_ui_command(context, status_port):
     rename_container(context.container_id, LOCAL_AGENT_CONTAINER_NAME + random_string(5))
     if context.container_id not in context.containers_id.keys():
         context.containers_id[context.container_id] = str(context.port)
+    threading.Event().wait(10)
```

Comment:
Isnt here the `10` to be the seconds from parameter?

Reply:
ok.